packs/journals/archetypes.json

```diff
@@ -1537,7 +1537,7 @@
+                "content": "<p>You shove your way through legions of foes, knock enemies on all sides to the ground, and deal massive blows to anyone or anything that comes near. You know that when it comes to taking down an enemy, it never hurts to have a bigger weapon.</p><p><strong>Additional Feats:</strong> <strong>4th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Slam Down], @UUID[Compendium.pf2e.feats-srd.Item.Vicious Swing]; <strong>12th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Crashing Slam]; <strong>14th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Brutal Finish], @UUID[Compendium.pf2e.feats-srd.Item.Unbalancing Sweep]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Mauler Dedication] <span style=\"float:right\">Feat 2</span></h2><p>You specialize in melee weapons that require two hands. You have familiarity with all melee weapons that require two hands to wield or have the two-hand trait. For the purposes of proficiency, you treat any of these that are martial weapons as simple weapons and any that are advanced weapons as martial weapons.</p><p>If you are at least an expert in such a weapon, you gain access to the critical specialization effect with that weapon.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Clear the Way] <span class=\"action-glyph\">2</span><span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.goFxIDlbWd8GN0kj inline hr]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Shoving Sweep] <span class=\"action-glyph\">r</span><span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.7p2tNqYHsg6u05cU inline hr]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Hammer Quake] <span class=\"action-glyph\">3</span><span style=\"float:right\">Feat 14</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.CR9NcAIPTT4oWSEy inline hr]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Avalanche Strike] <span class=\"action-glyph\">3</span><span style=\"float:right\">Feat 16</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.SGgK4BoUooA0HhTj inline hr]</p>",
```

Comment:
This one looks like it removed some embeds?

Reply:
You are correct, 

The dedication feat was embedded - We don't add those on the account that those feats have a link to the journal entry.